index.html

```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form>
+                <input type="text" maxlength="30" placeholder="Full name" required>
+                <input type="email" placeholder="Email adress" required>
+                <textarea maxlength="500" placeholder="Enter text here" required></textarea>
+                <button name="submit">Get in touch</button>
```

Comment:
- [ ] Please use the same button effects as mentioned in Figma Design your effects are not the same as Figma design check below :point_down: 
![Screenshot from 2023-02-21 21-02-33](https://user-images.githubusercontent.com/45226848/220401875-a695febf-240b-44cc-892d-60afa8bceb93.png)

- [ ] Kindly fix these styles on your end. :slightly_smiling_face: